README.md

```diff
@@ -23,9 +23,53 @@ Or download a prebuilt binary from [releases](./releases).
 ```bash
 protoc 
   --typescript-http_out [OUTPUT DIR] \
+  --typescript-http_opt use_enum_numbers=true,use_multi_line_comment=true
   [.proto files ...]
 ```
 
+#### Support options
+
+```ts
+// UseProtoNames controls the casing of generated field names.
+// If set to true, fields will use proto names (typically snake_case).
+// If omitted or set to false, fields will use JSON names (typically camelCase).
+use_proto_names: bool
+
+// UseEnumNumbers emits enum values as numbers.
+use_enum_numbers: bool
+
+// The method names of service methods naming case.
+// Only work when `UseEnumNumbers=true`
+// opt:
+// camelcase: convert name to lower camel case like `camelCase`
+// pascalcase: convert name to pascalcase like `PascalCase`
+// default is pascalcase
+enum_field_naming: string
+
+// Generate comments as multiline comments.
+// multiline comments: /** ... */
+// single line comments: // ...
+use_multi_line_comment: bool
+
+// force add `undefined` to message field.
+// default true
+force_message_field_undefinable: bool
+
+// If set to true, body will be JSON.stringify before send
+// default true
+use_body_stringify: bool
+
+// The method names of service methods naming case.
+// opt:
+// camelcase: convert name to lower camel case like `camelCase`
+// pascalcase: convert name to pascalcase like `PascalCase`
+// default is pascalcase
+service_method_naming: 'camelcase' | 'pascalcase'
+
+// If set to true, field int64 and uint64 will convert to string
+force_long_as_string: bool
```

Comment:
Should this just be called `long_as_string` instead? Not sure where the force comes from ;)

Reply:
This name is borrowed from: https://github.com/stephenh/ts-proto
<img width="906" alt="image" src="https://github.com/einride/protoc-gen-typescript-http/assets/37169906/b895913b-959a-4bc8-988a-a4e36c5ce961">

I would like to borrow from the existing naming rather than define a new one.

Of course, if you really don't like it, you can change it.